### PR TITLE
[ Ent-Corda ] Add chart for pki-node

### PR DESCRIPTION
**Pull request by deepakkumardbd**
**Changelog**
- Add chart for pki for node

 

**Reviewed by**
@jagpreetsinghsasan @suvajit-sarkar 

 

**Linked issue**
#898

### CODE REVIEW

File: platforms/corda-ent/charts/generate-pki/templates/configmap.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
   name: {{ .Values.nodeName }}-pkitool-conf
   namespace: {{ .Values.metadata.namespace }}
   labels:
-    app.kubernetes.io/name: {{ .Values.nodeName }}-cenm-conf
+    app.kubernetes.io/name: {{ .Values.nodeName }}-pkitool-conf
```
Comment (jagpreetsinghsasan) [inline platforms/corda-ent/charts/generate-pki/templates/configmap.yaml:8]: Please change the corresponding check for generate-pki too in roles/setup/signer

File: platforms/corda-ent/charts/pki-node/templates/job.yaml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+              value: {{ .Values.acceptLicense }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/bin/bash", "-c"]
+          args:
+          - |-
+            rm -r DATA/done.txt
+            mkdir  DATA/node
+            time java -Xmx{{ .Values.cordaJarMx }}G -jar {{ .Values.jarPath }}/pkitool.jar --config-file {{ .Values.configPath }}/pki.conf --ignore-missing-crl
+            #creating a dummy file to perform check if last line is executed or not.
+            touch DATA/done.txt
```
Comment (jagpreetsinghsasan) [inline platforms/corda-ent/charts/pki-node/templates/job.yaml:41]: Is this required?

Comment (deepakkumardbd) [reply platforms/corda-ent/charts/pki-node/templates/job.yaml:41]: Yes as this will be checked in next container.

**Comment by sownak** [comment]
@all-contributors Lets add @deepakkumardbd for great code

**Pull request merged**